boards/xtensa/esp32s2/esp32s2.dts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32s2.dtsi>
+
+/ {
+	model = "esp32s2";
+	compatible = "espressif,esp32s2";
+
+	aliases {
+		uart-0 = &uart0;
+		i2c-0 = &i2c0;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <ESP32_CLK_CPU_40M>;
```

Comment:
Why you didn't use PLL settings, is the clock_driver not compatible with ESP32-S2?